gammapy/detect/test_statistics.py

```diff
@@ -323,13 +323,22 @@ def compute_ts_map(counts, background, exposure, kernel, mask=None, flux=None,
     ----------
     [Stewart2009]_
     """
-    from scipy.ndimage.morphology import binary_erosion
     from time import time
     t_0 = time()
 
     assert counts.shape == background.shape
     assert counts.shape == exposure.shape
 
+    # in some maps there are pixels, which have exposure, but zero
+    # background, which doesn't make sense and causes the TS computation
+    # to fail, this is a temporary fix
+    mask_ = np.logical_and(background == 0,  exposure > 0)
+    if mask_.any():
+        log.warn('There are pixels in the data, that have exposure, but zero '
+                 'background, which can cause the ts computation to fail.'
+                 'Setting exposure of this pixels to zero.')
```

Comment:
Add space in the string on the line before. And on this line: "this pixels" -> "these pixels".